src/client/app/components/ReviewBubbles.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReviewBubble from './ReviewBubble.jsx';
+
+const ReviewBubbles = (props) => {
+  const bubbles = [];
+  const count = props.rating
+  for (let i = 0; i < count; i++) {
+    bubbles.push(<ReviewBubble />);
+  }
+  return (
+    <span>{bubbles}</span>
```

Comment:
You can push an empty bubble for the remaining, and then after that you'd just have to implement the half bubbles if you wanted to. But even solid bubbles plus (5 - solid) empties looks really good